### ISSUE TITLE
Cannot change line height using Line Height Adjuster

### ISSUE THREAD

**Issue by edsilv** [UniversalViewer/universalviewer#755]
https://chrome.google.com/webstore/detail/line-height-adjuster/nggicopcmcgpnpjbibppkjadniipeppa/related?hl=en-GB

**Comment by edsilv** [comment]
Tried using AlphaText to change the lineheights, but it only seems to work for certain pieces of text:

![image](https://user-images.githubusercontent.com/1334393/93821982-62503780-fc57-11ea-8cc6-6cb328ff6b1b.png)

I think it's quite out of date though, was last updated in 2017

**Comment by LlGC-szw** [comment]
All issues will be triaged for further investigation or closure by the 28 September 2023. If your issue is still relevant and would like for it be investigated further please comment by 14 September 2023.
